package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@moxy/react-modal",
+  "version": "0.0.0",
+  "description": "A modal, built on top of react-modal, that also fixes the scrollable body in iOS",
+  "author": {
+    "name": "Ivo Lima Silva",
+    "email": "ivolimasilva@outlook.com",
+    "url": "http://ivolimasilva.xyz"
+  },
+  "license": "MIT",
+  "main": "lib/index.js",
+  "module": "es/index.js",
+  "files": [
+    "lib",
+    "es"
+  ],
+  "scripts": {
+    "build:commonjs": "BABEL_ENV=commonjs babel src -d lib",
+    "build:es": "BABEL_ENV=es babel src -d es",
+    "build": "npm run build:commonjs && npm run build:es",
+    "lint": "eslint --ignore-path .gitignore .",
+    "test": "jest",
+    "prerelease": "npm t && npm run lint && npm run build",
+    "release": "standard-version",
+    "postrelease": "git push --follow-tags origin master && npm publish"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/moxystudio/react-modal.git"
+  },
+  "dependencies": {
+    "prop-types": "^15.7.2",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
Should both `react` and `react-dom` be peerDependencies?

We can add them to the devDependencies.

Reply:
Correct, just did that :+1: thanks!